components/Header/Header.jsx

```diff
@@ -22,8 +22,8 @@ export default function Header() {
                     </Link>
                     <nav id="navbar" className={`${styles.flexContainer} ${styles.navbar}`}>
                             <Link to="/">Home</Link>
-                            <Link to="docs">Documentation</Link>
-                            <Link to="./docs/api_explorer">API Explorer</Link>
+                            <Link to="docs/">Documentation</Link>
+                            <Link to="api-explorer/">API Explorer</Link>
```

Comment:
Isn't it `/docs/` and `/api-explorer/` instead of `docs/` and `api-explorer/`?

Reply:
/ in front comes automatically so no need to add it to the link